### PR TITLE
Sprint 8

### PR DESCRIPTION
**Pull request by raschdiaz**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -1,17 +0,0 @@
-{
```
Comment (zdmc23) [inline .vscode/launch.json]: Please do not include .vscode files in the PR

File: languages/tr.json [hunk 2]
```diff
@@ -0,0 +1,177 @@
+{
```
Comment (zdmc23) [inline languages/tr.json]: Please pull the latest bc we recently merged a PR for Turkish language support

**Comment by zdmc23** [comment]
@raschdiaz Please also resolve conflicts for:

screens/Group/GroupDetailScreen.js
screens/LoginScreen.js 

I don't mind resolving files that have a few conflicts, but those 2 files have double digit amount of conflicts.

File: .vscode/launch.json (hunk 1 above)
Comment (raschdiaz) [reply .vscode/launch.json]: Hi @zdmc23 , this .vscode files appear in the changes to indicate their elimination. At this moment this files does not exist in my current branch (sprint-8).

File: languages/tr.json (hunk 2 above)
Comment (raschdiaz) [reply languages/tr.json]: Several minutes ago i was updated the PR with this fixes.

**Pull request merged**